source/code/scxcorelib/pal/scxfile.cpp

```diff
@@ -576,7 +576,7 @@ namespace SCXCoreLib {
         \returns    Complete path of newly created file.
 
      */
-    SCXFilePath SCXFile::CreateTempFile(const std::wstring& fileContent) {
+    SCXFilePath SCXFile::CreateTempFile(const std::wstring& fileContent, const std::string& tmpDir /* = "/tmp/" */) {
```

Comment:
wstring.  It looks like you didn't implement the comments in earlier E-Mail.  You need to implement this prior to checkin.